README.md

```diff
@@ -1,216 +1,66 @@
-# GitHub Settings
+# GitHub Repository Settings
 
-<!--status-badges start -->
+This GitHub App syncs repository settings defined in `.github/settings.yml` to GitHub, enabling Pull Requests for repository settings.
 
-[![Powered by Vercel][vercel-badge]][vercel-link]
+<!--status-badges start -->
 
 [![Node CI Workflow Status][github-actions-ci-badge]][github-actions-ci-link]
 [![Renovate][renovate-badge]][renovate-link]
 
 <!--status-badges end -->
 
-This GitHub App syncs repository settings defined in `.github/settings.yml` to GitHub, enabling Pull Requests for repository settings.
+## Table of Contents
+
+* [Usage](#usage)
+  * [Install](#install)
+    * [Hosted GitHub.com App](#hosted-githubcom-app)
+    * [Self-Hosted App](#self-hosted-app)
+  * [Configuration](#configuration)
+* [Security Implications](#security-implications)
```

Comment:
I usually leave out a TOC these days, because GitHub has its own build into the UI now, and unless updating a TOC in a markdown file is automated it tends to get out of sync with the actually sections.

Reply:
thats fair. my reason for including is usually more for drawing attention to sections below the fold. i've known about the built-toc, but it is so hidden that i actually forgot it is even there, so i dont know that it accomplishes that goal. 

updating this one is fairly automatic because of the `generate:md` script that uses remark to update. i dont have a way to fully automate in mind, but i've wanted to investigate a way to fail verification if running the generate script would result in a diff.

open to more feedback on this, but lets address after getting his iteration merged